src/README.md

```diff
@@ -18,62 +18,30 @@ actions:
-    link: /reference/zilla.yaml/vault.html
+  - title: Get Started
+    icon: play
+    details: Install Zilla, connect it to your Kafka Cluster, and define your first API endpoints.
+    link: /get-started/install/
 
-  - title: Guards
-    icon: shield-halved
-    details: Each configured guard represents a security checkpoint for one or more bindings based on a specific implementation type.
-    link: /reference/zilla.yaml/guard.html
-
-  - title: Proxies
+  - title: Todo App Example
     icon: diagram-project
-    details: Design and configure your API Endpoints such as REST API, SSE and more.
-    link: /guides/kafka-proxies/rest-proxy.html
-
-  - title: Amazon MSK
-    icon: database
-    details: Learn how to deploy a more secure Amazon MSK with Aklivity Public MSK Proxy.
-    link: /get-started/connecting-to-kafka/amazon-msk.html
-
-  - title: Aiven
-    icon: server
-    details: Connect to Aiven Kafka from Zilla.
-    link: /get-started/connecting-to-kafka/aiven.html
-
-  - title: Confluent Cloud
-    icon: cloud
-    details: Connect to Confluent Cloud from Zilla.
-    link: /get-started/connecting-to-kafka/confluent-cloud.html
-
-  - title: Redpanda
-    icon: bars-staggered
-    details: Connect to Redpanda from Zilla.
-    link: /get-started/connecting-to-kafka/redpanda.html
+    details: Create a Todo application using a CQRS design pattern that's backed by Apache Kafka and Zilla.
+    link: /examples/todo-app/build.html
 
-copyright: false
----
+  - title: Kafka Proxies
+    icon: arrows-left-right-to-line
+    details: Define REST, SSE, gRPC and MQTT endpoints that map to Kafka topic streams.
+    link: /guides/kafka-proxies/rest-proxy.html
 
+  - title: Reference
+    icon: book
+    details: Learn about Zilla command and configuration properties.
+    link: /reference/
 
-**Zilla** is open-source software connecting web and mobile applications to event-driven microservices using standard protocols, such as HTTP, Server-Sent Events and Kafka.
 
-Zilla is designed on the fundamental principle that _every data flow is a stream_, and that streams can be composed together to create efficient protocol transformation pipelines. This concept of a stream holds at both the network level for communication protocols and also at the application level for data processing.
+footer: <span style="display:flex;align-items:center"><a href="https://www.aklivity.io"><img src="aklivity-logo@2x.png" height="30" alt="aklivity" style="padding-right:25px"></a> <a href="https://github.com/aklivity/zilla"><i class="fa-brands fa-github" style="font-size:22px;padding-right:6px"></i></a> <a href="https://www.linkedin.com/company/aklivity/"><i class="fa-brands fa-linkedin" style="font-size:22px;padding-right:6px"></i></a> <a href="https://www.aklivity.io/slack"><i class="fa-brands fa-slack" style="font-size:25px;padding-right:6px"></i></a> <a href="https://www.twitter.com/aklivityinc"><i class="fa-brands fa-twitter" style="font-size:22px"></i></a></span>
```

Comment:
This will only update the homepage footer. I added a comment for moving this into the theme config.